access-control-app/int-test/cmr/access_control/int_test/permission_check_test.clj

```diff
@@ -920,6 +921,48 @@
             "user1" ["update" "delete"]
             "user2" []))))))
 
+(deftest permission-get-and-post-request-test
+  (let [token (e/login-guest (u/conn-context))
+        save-basic-collection (fn [short-name]
+                                  (u/save-collection {:entry-title (str short-name " entry title")
+                                                      :short-name short-name
+                                                      :native-id short-name
+                                                      :provider-id "PROV1"}))
+        coll1 (save-basic-collection "coll1")
+        coll2 (save-basic-collection "coll2")
+        coll3 (save-basic-collection "coll3")
+        coll4 (save-basic-collection "coll4")
+        get-coll-permissions #(get-permissions :guest coll1 coll2 coll3 coll4)
+        base-url "http://localhost:3011/permissions"
+        get-url (str base-url
+                  "?user_type=guest"
+                  "&concept_id=" coll1
+                  "&concept_id=" coll2
+                  "&concept_id=" coll3
+                  "&concept_id=" coll4)
+        post-data-body (str "{ \"user_type\" : \"guest\", \"concept_id\" : ["
+                         "\"" coll1 "\", "
+                         "\"" coll2 "\", "
+                         "\"" coll3 "\", "
+                         "\"" coll4 "\""
+                         "] }")]
+
+    (testing "permissions endpoint allows get request"
+      (let [get-response (shell/sh "curl" "-g" "-i" get-url)
```

Comment:
There should be test helpers that use a http client that you can use to make calls, instead of curls.

Reply:
YOu also do not need a GET test for this ticket, only POST.  You could piggy back on another test that already does a get, and simply repeat the test again for POST

Reply:
Removed the GET test and modified the POST to use the http client.